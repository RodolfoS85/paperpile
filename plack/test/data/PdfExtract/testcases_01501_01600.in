---
file: ../../../PDFREPOSITORY/001501.pdf
---
file: ../../../PDFREPOSITORY/001502.pdf
---
file: ../../../PDFREPOSITORY/001503.pdf
---
file: ../../../PDFREPOSITORY/001504.pdf
---
file: ../../../PDFREPOSITORY/001505.pdf
---
file: ../../../PDFREPOSITORY/001506.pdf
---
file: ../../../PDFREPOSITORY/001507.pdf
---
file: ../../../PDFREPOSITORY/001508.pdf
---
file: ../../../PDFREPOSITORY/001509.pdf
---
file: ../../../PDFREPOSITORY/001510.pdf
---
file: ../../../PDFREPOSITORY/001511.pdf
---
file: ../../../PDFREPOSITORY/001512.pdf
---
file: ../../../PDFREPOSITORY/001513.pdf
---
file: ../../../PDFREPOSITORY/001514.pdf
---
file: ../../../PDFREPOSITORY/001515.pdf
---
file: ../../../PDFREPOSITORY/001516.pdf
---
file: ../../../PDFREPOSITORY/001517.pdf
---
file: ../../../PDFREPOSITORY/001518.pdf
---
file: ../../../PDFREPOSITORY/001519.pdf
---
file: ../../../PDFREPOSITORY/001520.pdf
---
file: ../../../PDFREPOSITORY/001521.pdf
---
file: ../../../PDFREPOSITORY/001522.pdf
---
file: ../../../PDFREPOSITORY/001523.pdf
---
file: ../../../PDFREPOSITORY/001524.pdf
---
file: ../../../PDFREPOSITORY/001525.pdf
---
file: ../../../PDFREPOSITORY/001526.pdf
---
file: ../../../PDFREPOSITORY/001527.pdf
---
file: ../../../PDFREPOSITORY/001528.pdf
---
file: ../../../PDFREPOSITORY/001529.pdf
---
file: ../../../PDFREPOSITORY/001530.pdf
---
file: ../../../PDFREPOSITORY/001531.pdf
---
file: ../../../PDFREPOSITORY/001532.pdf
---
file: ../../../PDFREPOSITORY/001533.pdf
---
file: ../../../PDFREPOSITORY/001534.pdf
---
file: ../../../PDFREPOSITORY/001535.pdf
---
file: ../../../PDFREPOSITORY/001536.pdf
---
file: ../../../PDFREPOSITORY/001537.pdf
---
file: ../../../PDFREPOSITORY/001538.pdf
---
file: ../../../PDFREPOSITORY/001539.pdf
---
file: ../../../PDFREPOSITORY/001540.pdf
---
file: ../../../PDFREPOSITORY/001541.pdf
---
file: ../../../PDFREPOSITORY/001542.pdf
---
file: ../../../PDFREPOSITORY/001543.pdf
---
file: ../../../PDFREPOSITORY/001544.pdf
---
file: ../../../PDFREPOSITORY/001545.pdf
---
file: ../../../PDFREPOSITORY/001546.pdf
---
file: ../../../PDFREPOSITORY/001547.pdf
---
file: ../../../PDFREPOSITORY/001548.pdf
---
file: ../../../PDFREPOSITORY/001549.pdf
---
file: ../../../PDFREPOSITORY/001550.pdf
---
file: ../../../PDFREPOSITORY/001551.pdf
---
file: ../../../PDFREPOSITORY/001552.pdf
---
file: ../../../PDFREPOSITORY/001553.pdf
---
file: ../../../PDFREPOSITORY/001554.pdf
---
file: ../../../PDFREPOSITORY/001555.pdf
---
file: ../../../PDFREPOSITORY/001556.pdf
---
file: ../../../PDFREPOSITORY/001557.pdf
---
file: ../../../PDFREPOSITORY/001558.pdf
---
file: ../../../PDFREPOSITORY/001559.pdf
---
file: ../../../PDFREPOSITORY/001560.pdf
---
file: ../../../PDFREPOSITORY/001561.pdf
---
file: ../../../PDFREPOSITORY/001562.pdf
---
file: ../../../PDFREPOSITORY/001563.pdf
---
file: ../../../PDFREPOSITORY/001564.pdf
---
file: ../../../PDFREPOSITORY/001565.pdf
---
file: ../../../PDFREPOSITORY/001566.pdf
---
file: ../../../PDFREPOSITORY/001567.pdf
---
file: ../../../PDFREPOSITORY/001568.pdf
---
file: ../../../PDFREPOSITORY/001569.pdf
---
file: ../../../PDFREPOSITORY/001570.pdf
---
file: ../../../PDFREPOSITORY/001571.pdf
---
file: ../../../PDFREPOSITORY/001572.pdf
---
file: ../../../PDFREPOSITORY/001573.pdf
---
file: ../../../PDFREPOSITORY/001574.pdf
---
file: ../../../PDFREPOSITORY/001575.pdf
---
file: ../../../PDFREPOSITORY/001576.pdf
---
file: ../../../PDFREPOSITORY/001577.pdf
---
file: ../../../PDFREPOSITORY/001578.pdf
---
file: ../../../PDFREPOSITORY/001579.pdf
---
file: ../../../PDFREPOSITORY/001580.pdf
---
file: ../../../PDFREPOSITORY/001581.pdf
---
file: ../../../PDFREPOSITORY/001582.pdf
---
file: ../../../PDFREPOSITORY/001583.pdf
---
file: ../../../PDFREPOSITORY/001584.pdf
---
file: ../../../PDFREPOSITORY/001585.pdf
---
file: ../../../PDFREPOSITORY/001586.pdf
---
file: ../../../PDFREPOSITORY/001587.pdf
---
file: ../../../PDFREPOSITORY/001588.pdf
---
file: ../../../PDFREPOSITORY/001589.pdf
---
file: ../../../PDFREPOSITORY/001590.pdf
---
file: ../../../PDFREPOSITORY/001591.pdf
---
file: ../../../PDFREPOSITORY/001592.pdf
---
file: ../../../PDFREPOSITORY/001593.pdf
---
file: ../../../PDFREPOSITORY/001594.pdf
---
file: ../../../PDFREPOSITORY/001595.pdf
---
file: ../../../PDFREPOSITORY/001596.pdf
---
file: ../../../PDFREPOSITORY/001597.pdf
---
file: ../../../PDFREPOSITORY/001598.pdf
---
file: ../../../PDFREPOSITORY/001599.pdf
---
file: ../../../PDFREPOSITORY/001600.pdf
