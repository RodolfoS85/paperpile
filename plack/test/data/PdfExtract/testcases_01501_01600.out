---
authors: 'Yang, Yuchen and Wang, Yu-Ping and Li, Kuo-Bin'
doi: 10.1186/1471-2105-9-S12-S4
title: 'MiRTif: a support vector machine-based microRNA target interaction filter'
---
authors: 'Yao, Zizhen and Barrick, Jeffrey and Weinberg, Zasha and Neph, Shane and Breaker, Ronald and Tompa, Martin and Ruzzo, Walter L.'
doi: 10.1371/journal.pcbi.0030126
title: 'A Computational Pipeline for High- Throughput Discovery of cis-Regulatory Noncoding RNA in Prokaryotes'
---
authors: 'Yap, Roland H. C.'
title: Parametric Sequence Alignment with Constraints
---
authors: 'Yasui, Dag and Miyano, Masaru and Cai, Shutao and Varga-Weisz, Patrick and Kohwi-Shigematsu, Terumi'
title: SATB1 targets chromatin remodelling to regulate genes over long distances
---
authors: 'Yeakley, Joanne M. and Fan, Jian-Bing and Doucet, Dennis and Luo, Lin and Wickham, Eliza and Ye, Zhen and Chee, Mark S. and Fu, Xiang-Dong'
title: Profiling alternative splicing on fiber-optic arrays
---
authors: 'Yeang, Chen-Hsiang and Vingron, Martin'
doi: 10.1186/1471-2105-7-332
title: A joint model of regulatory and metabolic networks
---
authors: 'Yeh, Iwei and Karp, Peter D. and Noy, Natalya F. and Altman, Russ B.'
title: 'Knowledge acquisition, consistency checking and concurrency control for Gene Ontology (GO)'
---
authors: 'Yelin, Rodrigo and Dahary, Dvir and Sorek, Rotem and Levanon, Erez Y. and Goldstein, Orly and Shoshan, Avi and Diber, Alex and Biton, Sharon and Tamir, Yael and Khosravi, Rami and Nemzer, Sergey and Pinner, Elhanan and Walach, Shira and Bernstein, Jeanne and Savitsky, Kinneret and Rotman, Galit'
doi: 10.1038/nbt808
title: Widespread occurrence of antisense transcription in the human genome
---
authors: 'Yeo, Gene and Burge, Christopher B.'
title: Maximum Entropy Modeling of Short Sequence Motifs with Applications to RNA Splicing Signals
---
authors: 'Yeo, Gene and Holste, Dirk and Kreiman, Gabriel and Burge, Christopher B'
title: Variation in alternative splicing across human tissues
---
authors: 'Yeo, Gene and Hoon, Shawn and Venkatesh, Byrappa and Burge, Christopher B.'
doi: 10.1073/pnas.0404901101
title: Variation in sequence and organization of splicing regulatory elements in vertebrate genes
---
authors: 'Yeo, Gene Wei-Ming'
doi: 10.1186/gb-2005-6-12-240
title: 'Splicing regulators: targets and drugs'
---
authors: 'Yeo, Gene W. and Van Nostrand, Eric and Holste, Dirk and Poggio, Tomaso and Burge, Christopher B.'
doi: 10.1073/pnas.0409742102
title: Identification and analysis of alternative splicing events conserved in human and mouse
---
authors: 'Yeung, K. Y. and Ruzzo, W. L.'
title: Principal component analysis for clustering gene expression data
---
authors: 'Yoffe, Aron M. and Prinsen, Peter and Gopal, Ajaykumar and Knobler, Charles M. and Gelbart, William M. and Ben-Shaul, Avinoam'
doi: 10.1073/pnas.0808089105
title: Predicting the sizes of large RNA molecules
---
authors: 'Yoon, Jeongah and Blumer, Anselm and Lee, Kyongbum'
title: An Algorithm for Modularity Analysis of Directed and Weighted Biological Networks Based on Edge-Betweenness Centrality
---
authors: 'Yoon, Sungroh and De Micheli, Giovanni'
doi: 10.1002/bdrc.20067
title: Computational Identification of MicroRNAs and Their Targets
---
authors: 'Yoshizawa, Satoko and Rasubala, Linda and Ose, Toyoyuki and Kohda, Daisuke and Fourmy, Dominique and Maenaka, Katsumi'
doi: 10.1038/nsmb890
title: Structural basis for mRNA recognition by elongation factor SelB
---
authors: 'Younes, Hakan L. S. and Kwiatkowska, Marta and Norman, Gethin and Parker, David'
title: 'Numerical vs. Statistical Probabilistic Model Checking: An Empirical Study'
---
authors: 'Young, Janet M and Shykind, Benjamin M and Lane, Robert P and Tonnes-Priddy, Lori and Ross, Joseph A and Walker, Megan and Williams, Eleanor M and Trask, Barbara J'
title: 'Odorant receptor expressed sequence tags demonstrate olfactory expression of over 400 genes, extensive alternate splicing and unequal expression levels'
---
authors: 'Yue, Kaizhi and Dill, Ken A.'
title: 'Inverse protein folding problem: Designing polymer sequences'
---
authors: 'Yue, Kaizhi and Dill, Ken A.'
title: Constraint-based assembly of tertiary protein structures from secondary structure elements
---
authors: 'Zampelli, Stphane and Mann, Martin and Deville, Yves and Backofen, Rolf'
title: Decomposition Techniques for Subgraph Matching
---
authors: 'Zeng, Fanfan and Yap, Roland H. C.'
title: Using Feature Generation and Feature Selection for Accurate Prediction of Translation Initiation Sites
---
authors: 'Zhang, Kaizhong'
title: Computing Similarity Between RNA Secondary Structures
---
authors: 'Zhang, Michael Q.'
title: A DISCRIMINATION STUDY OF HUMAN CORE-PROMOTERS
---
authors: '-F. ZHANG, Xiang H. and Arias, Mauricio A. and Ke, Shengdong and Chasin, Lawrence A.'
doi: 10.1261/rna.1498509
title: Splicing of designer exons reveals unexpected complexity in pre-mRNA splicing
---
authors: '-F. Zhang, Xiang H. and Chasin, Lawrence A.'
doi: 10.1073/pnas.0603042103
title: Comparison of multiple vertebrate genomes reveals the birth and evolution of human exons
---
authors: 'Zhang, Xiang H-F. and Chasin, Lawrence A.'
doi: 10.1101/gad.1195304
title: Computational definition of sequence motifs governing constitutive exon splicing
---
authors: 'Zhang, Wenyi and Chu, Xin and Tong, Qin and Cheung, Joseph Y. and Conrad, Kathleen and Masker, Kathryn and Miller, Barbara A.'
doi: 10.1074/jbc.M300298200
title: A Novel TRPM2 Isoform Inhibits Calcium Influx and Susceptibility to Cell Death
---
authors: 'Zhang, M. Q.'
title: Identification of protein coding regions in the human genome by quadratic discriminant analysis
---
authors: '-F. Zhang, Xiang H. and Kangsamaksin, Thaned and Chao, Mann S. P. and Banerjee, Joydeep K. and Chasin, Lawrence A.'
doi: 10.1128/MCB.25.16.7323-7332.2005
title: Exon Inclusion Is Dependent on Predictable Exonic Splicing Enhancers
---
authors: 'Zhang, Jinfeng and Kou, S. C. and Liu, Jun S.'
doi: 10.1063/1.2736681
title: Biopolymer structure simulation and optimization via fragment regrowth Monte Carlo
---
authors: 'Zhang, Michael Q.'
title: 'Large-Scale Gene Expression Data Analysis: A New Challenge to Computational Biologists'
---
authors: '-F. Zhang, Xiang H. and Leslie, Christina S. and Chasin, Lawrence A.'
doi: 10.1016/j.ymeth.2005.07.011
title: Computational searches for splicing signals
---
authors: 'Zhang, Xiang H-F. and Leslie, Christina S. and Chasin, Lawrence A.'
doi: 10.1101/gr.3217705
title: Dichotomous splicing signals in exon flanks
---
authors: 'Zhang, Yang and Skolnick, Jeffrey'
doi: 10.1073/pnas.0407152101
title: The protein structure prediction problem could be solved using the current PDB library
---
authors: 'Zhang, Hangning and Sreenivasulu, Nese and Weschke, Winfriede and Stein, Nils and Rudd, Stephen and Radchuk, Volodymyr and Potokina, Elena and Scholz, Uwe and Schweizer, Patrick and Zierold, Uwe and Langridge, Peter and Varshney, Rajeev K. and Wobus, Ulrich and Graner, Andreas'
doi: 10.1111/j.1365-313X.2004.02209.x
title: Large-scale analysis of the barley transcriptome based on expressed sequence tags
---
authors: 'Zhao, Lijun and Cox, Arthur G. and Ruzicka, Jan A. and Bhat, Ajita A. and Zhang, Weiqing and Taylor, Ethan Will'
title: Molecular modeling and in vitro activity of an HIV-1-encoded glutathione peroxidase
---
authors: 'Zheng, Zhi-Ming'
doi: 10.1159/000077096
title: Regulation of Alternative RNA Splicing by Exon Definition and Exon Sequences in Viral and Mammalian Gene Expression
---
authors: 'Zhu, Jun and Liu, Jun S. and Lawrence, Charles E.'
title: Bayesian adaptive sequence alignment algorithms
---
authors: 'Zhu, J. and Zhang, M. Q.'
title: 'CLUSTER, FUNCTION AND PROMOTER: ANALYSIS OF YEAST EXPRESSION ARRAY'
---
authors: 'Zhu, Jian and Zhang, Michael Q.'
title: 'SCPD: a promoter database of the yeast Saccharomyces cerevisiae'
---
authors: 'Zhuang, Yonglong and Ma, Fei and Li-Ling, Jesse and Xu, Xiaofeng and Li, Yanda'
title: Comparative analysis of amino acid usage and protein length distribution between alternatively and non-alternatively spliced genes across six eukaryotic genomes
---
authors: 'Zhuo, Degen and Madden, Richard and Elela, Sherif Abou and Chabot, Benoit'
doi: 10.1073/pnas.0604777104
title: Modern origin of numerous alternatively spliced human introns from tandem arrays
---
authors: 'Zilberstein, Chaya BEN-ZAKEN and ZIV-UKELSON, Michal and Pinter, Ron Y. and Yakhini, Zohar'
title: A High-Throughput Approach for Associating MicroRNAs with Their Activity Conditions
---
authors: 'Zucconi, Adriana and Panni, Simona and Paoluzi, Serena and Castagnoli, Luisa and Dente, Luciana and Cesareni, Gianni'
title: Domain repertoires as a tool to derive protein recognition rules
---
authors: 'Zuker, Michael and Jacobson, Ann B.'
title: Using reliability information to annotate RNA secondary structures
---
authors: 'Zuker, Michael and Jaeger, John A. and Turner, Douglas H.'
title: A comparison of optimal and suboptimal RNA secondary structures predicted by free energy minimization with structures determined by phylogenetic comparison
---
authors: 'Zuker, Michael'
title: Calculating nucleic acid secondary structure
---
authors: 'Zupicich, Joel and Brenner, Steven E and Skarnes, William C'
title: Computational prediction of membrane-tethered transcription factors
---
authors: 'Naya, Francisco J and Olson, Eric'
title: 'MEF2: a transcriptional target for signaling pathways controlling skeletal muscle growth and differentiation'
---
authors: 'Burd, Christopher G. and Strochlic, Todd I. and Gangi Setty, Subba R.'
doi: 10.1016/j.tcb.2004.10.004
title: 'Arf-like GTPases: not so Arf-like after all'
---
authors: 'Beer, Hans-Dietmar and Wohlfahrt, Gerd and Schmid, Rolf D. and Mccarthy, John E. G.'
title: The folding and activity of the extracellular lipase of Rhizopus oryzae are modulated by a prosequence
---
authors: 'Lecaille, Fabien and Weidauer, Enrico and Juliano, Maria A. and Bromme, Dieter and Lalmanach, Gilles'
title: Probing cathepsin K activity with a selective substrate spanning its active site
---
authors: 'Marsh, Derek and Horvath, Laszlo I.'
title: 'Structure, dynamics and composition of the lipid-protein interface. Perspectives from spin-labelling'
---
authors: 'Ovaska, Kristian and Laakso, Marko and Hautaniemi, Sampsa'
doi: 10.1186/1756-0381-1-11
title: Fast Gene Ontology based clustering for microarray experiments
---
authors: 'McEachin, Richard C and Keller, Benjamin J and Saunders, Erika FH and McInnis, Melvin G'
doi: 10.1186/1756-0381-1-2
title: Modeling gene-by-environment interaction in comorbid depression with alcohol use disorders via an integrated bioinformatics approach
---
authors: 'Hughey, Richard and Krogh, Anders'
title: 'Hidden Markov models for sequence analysis: extension and analysis of the basic method'
---
authors: 'Smalheiser, Neil R'
doi: 10.1186/1745-6150-3-44
title: Synaptic enrichment of microRNAs in adult mouse forebrain is related to structural features of their precursors
---
authors: 'Ferrari, Maud C. O. and Chivers, Douglas P.'
doi: 10.1098/rsbl.2008.0641
title: Latent inhibition of predator recognition by embryonic amphibians
---
authors: 'Hsueh, Chia-Hsiang and Chen, Wen-Pin and Lin, Jiunn-Lee and Tsai, Chia-Ti and Liu, Yen-Bin and Juang, Jyh-Ming and Tsao, Hsuan-Ming and Su, Ming-Jai and Lai, Ling-Ping'
doi: 10.1186/1423-0127-16-23
title: Distinct functional defect of three novel Brugada syndrome related cardiac sodium channel mutations
---
authors: 'Keller, Andreas and Backes, Christina and Al-Awadhi, Maher and Gerasch, Andreas and Küntzer, Jan and Kohlbacher, Oliver and Kaufmann, Michael and Lenhof, Hans-Peter'
doi: 10.1186/1471-2105-9-552
title: 'GeneTrailExpress: a web-based pipeline for the statistical evaluation of microarray experiments'
---
authors: 'Ge, Xiaomeng and Chen, Weihua and Song, Shuhui and Wang, Weiwei and Hu, Songnian and Yu, Jun'
doi: 10.1186/1471-2229-8-114
title: Transcriptomic profiling of mature embryo from an elite super-hybrid rice LYP9 and its parental lines
---
authors: 'Cooper, Stephen and Shedden, Kerby'
title: Microarray analysis of gene expression during the cell cycle
---
authors: 'Vadakkan, Kunjumon I and Li, Baoxiang and De Boni, Umberto'
doi: 10.1186/1475-9268-5-1
title: Trend towards varying combinatorial centromere association in morphologically identical clusters in Purkinje neurons
---
authors: 'Villanueva, Melissa and Thornley, Keith and Augustine, George J. and Wightman, R. Mark'
doi: 10.1007/s11068-007-9015-2
title: Synapsin II negatively regulates catecholamine release
---
authors: 'Krstevska-Konstantinova, M and Kocova, M and Gucev, Z and Sukarova-Angelovska, E'
title: 'Premature thelarche in Macedonia: a three-year follow-up'
---
authors: 'Hijova, E and Chmelarova, A'
title: Short chain fatty acids and colonic health
---
authors: 'Alabakovska, SB and Labudovic, DD and Tosheska, KN and Spiroski, MZ and Todorova, BB'
title: Low density lipoprotein subclass distribution in children with diabetes mellitus
---
authors: 'Blunt, John W. and Copp, Brent R. and Hu, Wan-Ping and Munro, Murray H. G. and Northcote, Peter T. and Prinsep, Michle R.'
doi: 10.1039/b603047p
title: Marine natural products
---
authors: 'Takeuchi, Toshifumi and Hishiya, Takayuki'
doi: 10.1039/b715737c
title: Molecular imprinting of proteins emerging as a tool for protein recognition
---
authors: 'Kreissl, S. and Uber, A. and Harzsch, S.'
doi: 10.1007/s00427-008-0216-1
title: 'Muscle precursor cells in the developing limbs of two isopods (Crustacea, Peracarida): an immunohistochemical study using a novel monoclonal antibody against myosin heavy chain'
---
authors: 'Mochizuki, Kazufumi and Sano, Hiroko and Kobayashi, Satoru and Nishimiya-Fujisawa, Chiemi and Fujisawa, Toshitaka'
title: Expression and evolutionary conservation of nanos-related genes in Hydra
---
authors: 'Salmon, D. and Hanocq-Quertier, J. and Paturiaux-Hanocq, F. and Pays, A. and Tebabi, P. and Nolan, D. P. and Michel, A. and Pays, E.'
title: Characterization of the ligand-binding site of the transferrin receptor in Trypanosoma brucei demonstrates a structural relationship with the N-terminal domain of the variant surface glycoprotein
---
authors: 'Hutschenreiter, Silke and Tinazli, Ali and Model, Kirstin and Tampe, Robert'
doi: 10.1038/sj.emboj.7600262
title: Two-substrate association with the 20S proteasome at single-molecule level
---
authors: 'Grossmann, Guido and Opekarova, Miroslava and Malinsky, Jan and Weig-Meckl, Ina and Tanner, Widmar'
doi: 10.1038/sj.emboj.7601466
title: Membrane potential governs lateral segregation of plasma membrane proteins and lipids in yeast
---
authors: 'van den Berg, Bert and Wain, Rachel and Dobson, Christopher M. and Ellis, R. John'
title: 'Macromolecular crowding perturbs protein refolding kinetics: implications for folding inside the cell'
---
authors: 'Sung, Mei-an and Fleming, Keiran and Chen, Ho An and Matthews, Stephen'
title: The solution structure of PapGII from uropathogenic Escherichia coli and its recognition of glycolipid receptors
---
authors: 'Khan, Haseeb A. and Arif, Ibrahim A. and Bahkali, Ali H. and Al Farhan, Ahmad H. and Al Homaidan, Ali A.'
title: 'Bayesian, Maximum Parsimony and UPGMA Models for Inferring the Phylogenies of Antelopes Using Mitochondrial Markers'
---
authors: 'Tehei, Moeava and Zaccai, Giuseppe'
doi: 10.1111/j.1742-4658.2007.05953.x
title: Adaptation to high temperatures through macromolecular dynamics by neutron scattering
---
authors: 'Gallmetzer, Martin and Meraner, Joachim and Burgstaller, Wolfgang'
title: Succinate synthesis and excretion by Penicillium simplicissimum under aerobic and anaerobic conditions
---
authors: 'Bali, D and Gourley, S and Kostyu, DD and Goel, N and Bruce, I and Bell, A and Walker, DJ and Tran, K and Zhu, DK and Costello, TJ and Amos, CI and Seldin, MF'
title: Genetic analysis of multiplex rheumatoid arthritis families
---
authors: 'Lee, Ki Won and Kang, Nam Joo and Kim, Jong Hun and Lee, Kyoung Mi and Lee, Dong Eun and Hur, Haeng Jeon and Lee, Hyong Joo'
doi: 10.1007/s12263-007-0067-9
title: Caffeic acid phenethyl ester inhibits invasion and expression of matrix metalloproteinase in SK-Hep1 human hepatocellular carcinoma cells by targeting nuclear factor kappa B
---
authors: 'van Ommen, Ben'
doi: 10.1007/s12263-007-0018-5
title: 'Personalized nutrition from a health perspective: luxury or necessity?'
---
authors: 'Usukura, Jiro and Nishizawa, Yuji and Shimomura, Atsushi and Kobayashi, Kazuto and Nagatsu, Toshiharu and Hagiwara, Masatoshi'
title: Direct imaging of phosphorylation-dependent conformational change and DNA binding of CREB by electron microscopy
---
authors: 'Souter, Vivienne L. and Parisi, Melissa A. and Nyholt, Dale R. and Kapur, Raj P. and Henders, Anjali K. and Opheim, Kent E. and Gunther, Daniel F. and Mitchell, Michael E. and Glass, Ian A. and Montgomery, Grant W.'
doi: 10.1007/s00439-006-0279-x
title: A case of true hermaphroditism reveals an unusual mechanism of twinning
---
authors: 'Paganin-Gioanni, Aurelie and Bellard, Elisabeth and Couderc, Bettina and Teissié, Justin and Golzio, Muriel'
title: Tracking in vitro and in vivo siRNA electrotransfer in tumor cells
---
authors: 'Keller, Alexander and Rödel, Mark-Oliver and Eduard Linsenmair, K. and Ulmar Grafe, T.'
doi: 10.1111/j.1365-2656.2008.01457.x
title: The importance of environmental heterogeneity for species diversity and assemblage structure in Bornean stream frogs
---
authors: 'Tyrka, Miros³aw and Perovic, Dragan and Wardyñska, Agnieszka and Ordon, Frank'
title: A new diagnostic SSR marker for selection of the Rym4/Rym5 locus in barley breeding
---
authors: 'Baylis, C. L.'
doi: 10.1111/j.1365-2672.2008.03859.x
title: Growth of pure cultures of Verocytotoxin-producing Escherichia coli in a range of enrichment media
---
authors: 'Peng, SHI-QING and Zhu, JIA-HONG and Li, HUI-LIANG and -MIN TIAN, W'
title: Cloning and characterization of a novel cysteine protease gene (HbCP1) from Hevea brasiliensis
---
authors: 'Su, Zhida and Cao, Li and Zhu, Yanling and Liu, Xiujie and Huang, Zhihui and Huang, Aijun and He, Cheng'
doi: 10.1242/jcs.03448
title: Nogo enhances the adhesion of olfactory ensheathing cells and inhibits their migration
---
authors: 'Alfredsson-Timmins, Jenny and Henningson, Frida and Bjerling, Pernilla'
doi: 10.1242/jcs.03457
title: The Clr4 methyltransferase determines the subnuclear localization of the mating-type region in fission yeast
---
authors: 'Troyo, Adriana and Fuller, Douglas O. and Calderón-Arguedas, Olger and Beier, John C.'
title: A geographical sampling method for surveys of mosquito larvae in an urban area using high-resolution satellite imagery
---
authors: 'Kargi, F.'
doi: 10.1111/j.1472-765X.2008.02537.x
title: Re-interpretation of the logistic equation for batch microbial growth in relation to Monod kinetics
---
authors: 'Santos, I. M. and Abrunhosa, L. and Venancio, A. and Lima, N.'
title: The effect of culture preservation techniques on patulin and citrinin production by Penicillium expansum Link
---
authors: 'Zaman, Shadia and Lippman, Soyeon I and Schneper, Lisa and Slonim, Noam and Broach, James R'
doi: 10.1038/msb.2009.2
title: Glucose regulates transcription in yeast through a network of signaling pathways
---
authors: 'Dodson, M. L. and Lloyd, R. Stephen'
title: 'Backbone dynamics of DNA containing 8-oxoguanine: importance for substrate recognition by base excision repair glycosylases'
---
authors: 'Abrescia, Nicola G. A. and Malinina, Lucy and Fernandez, Luzimar G. and Huynh-Dinh, Tam and Neidle, Stephen and Subirana, Juan A.'
title: Structure of the oligonucleotide d(CGTATATACG) as a site-specific complex with nickel ions
